interfaces/Glitter/templates/static/stylesheets/glitter.css

```diff
@@ -1840,6 +1845,15 @@ input[name="nzbURL"] {
     padding-top: 5px;
 }
 
+.container-unlimited-width .container {
+    width: calc(100% - 60px);
+}
+
+.queue-table .table-header-progress-smaller {
+    width: 25%;
+}
```

Comment:
This should not be here, because we only support extra columns when the screen size is >1200pixels, otherwise everything is smashed together.
That's why it's also listed below in the `Dynamic sizing` section.